### DISCUSSION TITLE
[Exercise-7]: Write in your own words about bind variables and share screenshots of your examples along with the source code.

### DISCUSSION THREAD

**Discussion by acrlakshman** [exaslate-courses/oracle-sql-plsql-b3#8] [General]
* Push changes to `0-exercices/<name>`
  * File name: `7-bind-variables.sql`

Ref.: https://www.youtube.com/watch?v=spzUsq7j7J4&list=PLF6_5RI6ENIlDi3knzcQGUiFc5-WdA_1W&index=3&t=317s

**Comment by Tharun1596** [comment]
bind variable acts as a communication between the sql and pl/sql
we need to use colon before code = :v_code

**Comment by srideviadapa0511** [comment]
![BINDING VARIABLES](https://github.com/exaslate-courses/oracle-sql-plsql-b3/assets/145452411/db8e44db-84fc-4e5b-8d94-ea9593ec3c14)

**Reply by acrlakshman** [reply]
@srideviadapa0511 , in line-2, please remove ":" before `V_BIND1`. You should only use `:` before a bind variable, whenever you are using it, but not during `declaration` step.

**Comment by srideviadapa0511** [comment]
![bind](https://github.com/exaslate-courses/oracle-sql-plsql-b3/assets/145452411/1696006b-0745-41d4-a64b-41ad108fe89c)

**Reply by srideviadapa0511** [reply]
Is this good sir ??

**Reply by acrlakshman** [reply]
@srideviadapa0511 , yes this is good. can you please replace `VARIABLE V_BIND1 VARCHAR(30)='SRIDEVI'` in `line-1` with `VARIABLE V_BIND1 VARCHAR2 = 'SRIDEVI'`

**Reply by srideviadapa0511** [reply]
ok sir 
![bind update](https://github.com/exaslate-courses/oracle-sql-plsql-b3/assets/145452411/3c86ddbe-1c28-4ec9-a7a0-646044444280)

**Reply by acrlakshman** [reply]
@srideviadapa0511 , excellent. Now you can see that the output is clean.

**Comment by cm2024jsp** [comment]
DECLARE
  emp_id NUMBER := 101; 
  emp_name VARCHAR2(50);
BEGIN
  SELECT first_name INTO emp_name FROM employees WHERE employee_id = emp_id;

END;

**Reply by acrlakshman** [reply]
@cm2024jsp nice work. I think you might have accidentally shared this answer here, instead of in exercise-4. But that' ok.
